### PR TITLE
Add Kafka config for prod env (DEV-119)

### PR DESCRIPTION
**Pull request by scflode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ to be reachable):
 - `SECRET_KEY_BASE`: The Phoenix secret. (generate via `mix phx.gen.secret`)
 - `HOSTNAME`: The hostname needed for WebSockets. (f.e. `localhost`)
 - `PROJECTORS` (optional): The projectors that should be active. Can either be `all` (default) or `local`.
+- `KAFKA_HOSTS`: The Kafka host(s). (e.g. `localhost:9092`); Can be an empty string if `local` is used.
```
Comment (scflode) [inline README.md:92]: This env vars needs to be set in addition.

Comment (head1328) [reply README.md:92]: Multiple hosts will be separated by comma?

Comment (scflode) [reply README.md:92]: Yes.

File: config/releases.exs [hunk 2]
```diff
@@ -28,6 +28,16 @@ config :dealog_backoffice, DealogBackoffice.EventStore,
 config :dealog_backoffice, :projection,
   projectors: String.to_existing_atom(System.get_env("PROJECTORS")) || :all
 
+kafka_hosts =
+  System.get_env("KAFKA_HOSTS") ||
+    raise """
+        environment variable KAFKA_HOSTS is missing.
+        For example: localhost:9092,localhost:9093,localhost:9094
```
Comment (scflode) [inline config/releases.exs:35]: @head1328 see here

**Pull request merged**